new/INS0984MLQM.xml

```diff
@@ -48,9 +48,13 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                     <placeName xml:lang="gez" xml:id="n1">መንበረ፡ ልዑል፡ ቅዱስ፡ ማርቆስ፡</placeName>
                     <placeName xml:lang="gez" corresp="#n1" type="normalized">Manbara Lǝʿul Qəddus Mārqos </placeName>
                     <country ref="wd:Q115">Ethiopia</country>
-                    <settlement ref="LOC4508Manbar"/> 
                 </place>
             </listPlace>
+            <div type="bibliography">
```

Comment:
thank you! we still sould need region and settlement, or the visualization would be with empty spaces and commas 
the church is in Addis Ababa (Sidist Kilo I believe)

Reply:
oh, sorry, that's true! I tried adding more information.